docker-compose.yml

```diff
@@ -11,4 +11,19 @@ services:
     ports:
       - ${DB_PORT}:5432
     volumes:
-      - ./docker-data/db:/var/lib/postgresql/data
+      - pgdata:/var/lib/postgresql/data
+
+  app:
```

Comment:
Like I mentioned in the PR comment, lets move this service to a separate file `docker-compose.prod.yml`